src/hooks/useAllContractsOwned.ts

```diff
@@ -0,0 +1,15 @@
+import { useSnapshot } from 'valtio'
+import ContractsNetworkStore from 'stores/ContractsStore'
+
+export default function () {
+  const { networks } = useSnapshot(ContractsNetworkStore)
+
+  const addressToTokenIds = Object.values(networks).reduce(
+    (res, network) => ({ ...res, ...network.addressToTokenIds }),
```

Comment:
Btw, what is `res`? Maybe we can have another name for `res`, isn't this is a `prev`?